From 221f9553e5094a092a4592812d67111e94d1f329 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Thu, 31 May 2018 15:53:13 +0200
Subject: CORS legacy: add missing string include

The cors_legacy.h file includes declarations using std::string, but
it is not declared due to missing #include <string>.

Also drop unneeded declarations in .cpp file.

Change-Id: I00df799f84a6c3530c2f12f1e52d24c7d9bd6bfd

diff --git a/services/network/public/cpp/cors/cors_legacy.cc b/services/network/public/cpp/cors/cors_legacy.cc
index 8f32ac3be64e..f01af63619b5 100644
--- a/services/network/public/cpp/cors/cors_legacy.cc
+++ b/services/network/public/cpp/cors/cors_legacy.cc
@@ -5,8 +5,6 @@
 #include "services/network/public/cpp/cors/cors_legacy.h"
 
 #include <algorithm>
-#include <string>
-#include <vector>
 
 #include "url/gurl.h"
 #include "url/url_util.h"
diff --git a/services/network/public/cpp/cors/cors_legacy.h b/services/network/public/cpp/cors/cors_legacy.h
index d2cdf026ca3a..dc9295a92c47 100644
--- a/services/network/public/cpp/cors/cors_legacy.h
+++ b/services/network/public/cpp/cors/cors_legacy.h
@@ -5,6 +5,7 @@
 #ifndef SERVICES_NETWORK_PUBLIC_CPP_CORS_CORS_LEGACY_H_
 #define SERVICES_NETWORK_PUBLIC_CPP_CORS_CORS_LEGACY_H_
 
+#include <string>
 #include <vector>
 
 #include "base/component_export.h"
-- 
2.14.1

